moped-editor/src/views/dev/LookupsView/settings.js

```diff
@@ -205,4 +205,22 @@ export const SETTINGS = [
       },
     ],
   },
+  {
+    key: "moped_project_roles",
+    label: "Roles",
```

Comment:
Would you place this between the entires for Phases and Tag, such that the nav buttons will be in alphabetical order? 📖 ✨

Reply:
🙏 🔡 ✅